tests/test_mixins.py

```diff
@@ -21,6 +21,11 @@ def test_checkout_branch(mocked_config):
     mocked_config.settings.dry_run = False
     mocked_config.settings.git_branch_name = expected_branch_name
 
+    mock_head = Mock()
+    mock_head.name = expected_branch_name
+
+    mocked_config.repo.heads = [mock_head]
```

Comment:
minor: I would not put this into 3 lines
```
mocked_config.repo.heads = [Mock(name=expected_branch_name)]

```

Reply:
This won't work, since if you initialize a mock object that way, it will be the mock's name and not the parameter you'd expect